### PR TITLE
Moving existing content from https://github.com/Ericsson/secure_coding_one_stop_shop_for_python

### PR DESCRIPTION
**Pull request by tommcd**
Initial version of Python guide

    - Delete docs/Secure-Coding-Guide-for-Python/CODEOWNERS
    - Delete CODEOWNERS
    - Delete docs/Secure-Coding-Guide-for-Python/.gitignore

### CODE REVIEW

File: docs/Secure-Coding-Guide-for-Python/CODEOWNERS
Comment (tommcd) [inline docs/Secure-Coding-Guide-for-Python/CODEOWNERS:1]: I think I should delete this file?

Comment (gkunz) [reply docs/Secure-Coding-Guide-for-Python/CODEOWNERS:1]: yes, I agree

File: docs/Secure-Coding-Guide-for-Python/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.code-workspace
```
Comment (gkunz) [inline docs/Secure-Coding-Guide-for-Python/.gitignore:1]: I think this file could also be removed.

Comment (tommcd) [reply docs/Secure-Coding-Guide-for-Python/.gitignore:1]: Removing for now. Maybe we can have a more complete .gitignore for python later, for example:
https://www.toptal.com/developers/gitignore?templates=python,virtualenv,pycharm,visualstudiocode

**Review by gkunz — COMMENTED** [review]
Thank you @tommcd! There are a few minor issues in the readme that the linter is complaining about. I marked those in the PR.

**Review by SecurityCRob — COMMENTED** [review]
@myteron and @gkunz , let me know when you are happy with this and we can merge to get things started

**Pull request merged**